### DISCUSSION TITLE
how to check version in web gui

### DISCUSSION THREAD

**Discussion by home4lab** [mikopbx/Core#383] [Q&A] [answered]
recently i've done upgrading miko pbx. then i want to check version is updated or not.

is anyone can show how to check version of miko pbx in web gui?
if not, is possible to add menu like about? so, in that menu we can see the version

**Comment by boffart** [comment, marked as answer]
<img width="716" alt="1" src="https://user-images.githubusercontent.com/5922739/182098114-41ec1d0a-9a52-4ae3-83ee-9bd33829e7be.png">

<img width="1339" alt="2" src="https://user-images.githubusercontent.com/5922739/182098118-9c98bb25-631f-4ec1-bb0a-a3ab1dab9183.png">

**Comment by home4lab** [comment]
oh, i see. thank you sir
